packages/wallets/src/AppContent.tsx

```diff
@@ -27,6 +32,21 @@ const AppContent: React.FC = () => {
         defineViewportHeight();
     }, []);
 
+    useEffect(() => {
+        if (!isSuccess) return;
+        if (isSubscribed) unsubscribe();
+        subscribe({
+            account: 'all',
+        });
+        return () => {
+            if (isSubscribed) unsubscribe();
+        };
+    }, [balanceData, isSubscribed, isSuccess, subscribe, unsubscribe]);
+
+    useEffect(() => {
+        setBalanceData({ ...balanceData });
```

Comment:
Yes @wojciech-deriv, that's true, but how do you propose we maintain only one subscription every time we call the `useSubscribedBalance` hook? 

I put the logic here, because I wanted to `subscribe` once, i.e. when `AppContents` mounts.